backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java

```diff
@@ -251,6 +258,49 @@ void testCreateAndGetAndUpdateAndDeleteEventType() {
         getEventTypes(appId, OK, 1);
     }
 
+    @Test
+    void testDefaultBehaviorGroupCRUD() {
+        // We need to persist a bundle, an app and two event types for this test.
```

Comment:
Where are the app and the event types?

Reply:
```suggestion
        // We need to persist a bundle.
```

Didn't use them in the end and forgot to update :D